downstream/modules/analytics/proc-add-roles-to-group.adoc

```diff
@@ -11,10 +11,10 @@ Adding roles to an existing group will grant new permissions to existing users i
 .Procedure
 
 . Log in to cloud.redhat.com
-. Click image:images/cog.png[] > *Settings*.
-. From the sidebar, navigate to *User Access* > *Groups*.
+. Click menu:image:images/cog.png[][Settings].
```

Comment:
I think this is actually calling in an image file and so should stay image:images/cog.png

Reply:
Change to image:/cog.png[] because the image path has been changed in analytics modules.